### ISSUE TITLE
Using Nearley without Node?

### ISSUE THREAD

**Issue by m00nr4bb1t** [kach/nearley#464]
Hello,

It says in the docs that to use in the browser, one can simply use script tags to include nearley.js and the compiled grammar.js. Does that mean the Nearley **parser** (not compiler) is completely standalone and all I need to use it is the nearley.js file (and, of course, the grammar file)? So, does that mean I don't need Node if I just wanted to use the parser in the browser?

+ I saw that there wasn't any sort of copyright notice in the nearley.js file; if I wanted to include it in my project without any strings attached (assuming that's possible) I assume I'd have to add one in..?

**Comment by kach** [comment]
Correct, the parser should work fine on its own without node. You should include the license in https://github.com/kach/nearley/blob/master/LICENSE.txt.
